alex/corpustools/voxforge-download.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+url=http://www.repository.voxforge1.org/downloads/SpeechCorpus/Trunk/Audio/Main/16kHz_16bit
```

Comment:
There's only English at this URL. I wasn't aware that there was audio in other languages because the English VoxForge website only links English data. I will make the language a parameter.
As for the frequencies, there is also 8kHz_16bit. Should I make this a (optional) parameter as well?